<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ggClassyMainWindow</class>
 <widget class="QMainWindow" name="ggClassyMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1258</width>
    <height>930</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Classy Architect</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="ggGraphicsView" name="mGraphicsView">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="renderHints">
       <set>QPainter::Antialiasing|QPainter::TextAntialiasing</set>
      </property>
      <property name="transformationAnchor">
       <enum>QGraphicsView::NoAnchor</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>3</number>
      </property>
      <item>
       <widget class="QPushButton" name="mOpenPushButton">
        <property name="text">
         <string>Open...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mSaveAsPushButton">
        <property name="text">
         <string>Save As...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mNewCollectionPushButton">
        <property name="text">
         <string>New Coll.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mNewClassPushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>New Class</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mNewClassBoxPushButton">
        <property name="text">
         <string>New Box</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mNewFramePushButton">
        <property name="text">
         <string>New Frame</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mDeleteCollectionPushButton">
        <property name="text">
         <string>Del Coll.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mDeleteClassPushButton">
        <property name="text">
         <string>Del Class</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mDeleteConnectionPushButton">
        <property name="text">
         <string>Del Conn.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mDeleteClassBoxPushButton">
        <property name="text">
         <string>Del Box</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mDeleteFramePushButton">
        <property name="text">
         <string>Del Frame</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mMoveTopPushButton">
        <property name="text">
         <string>Top</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mMoveUpPushButton">
        <property name="text">
         <string>Up</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mMoveDownPushButton">
        <property name="text">
         <string>Down</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mMoveBottomPushButton">
        <property name="text">
         <string>Bottom</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ggColorWheelWidget" name="mColorWheelWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ggColorChannelWidget" name="mColorChannelWidgetV" native="true">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ggColorChannelWidget" name="mColorChannelWidgetA" native="true">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ggColorChannelWidget" name="mColorChannelWidgetR" native="true">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ggColorChannelWidget" name="mColorChannelWidgetG" native="true">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ggColorChannelWidget" name="mColorChannelWidgetB" native="true">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>2</number>
         </property>
         <item row="0" column="1">
          <widget class="QToolButton" name="mAlignTC">
           <property name="text">
            <string>x</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QToolButton" name="mAlignTR">
           <property name="text">
            <string>x</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QToolButton" name="mAlignCL">
           <property name="text">
            <string>x</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QToolButton" name="mAlignCC">
           <property name="text">
            <string>x</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QToolButton" name="mAlignCR">
           <property name="text">
            <string>x</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QToolButton" name="mAlignBL">
           <property name="text">
            <string>x</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QToolButton" name="mAlignBC">
           <property name="text">
            <string>x</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QToolButton" name="mAlignBR">
           <property name="text">
            <string>x</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QToolButton" name="mAlignTL">
           <property name="text">
            <string>x</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>48</width>
          <height>88</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="mZoomLabel">
        <property name="text">
         <string>Zoom:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="mZoomComboBox">
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="currentIndex">
         <number>3</number>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::NoInsert</enum>
        </property>
        <item>
         <property name="text">
          <string>10%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>50%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>100%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>200%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>500%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1000%</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mZoomResetPushButton">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mZoomFitPushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Fit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1258</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ggGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>BaseGraphics/ggGraphicsView.h</header>
   <slots>
    <slot>SetZoom100()</slot>
    <slot>SetZoomFit()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>ggColorWheelWidget</class>
   <extends>QWidget</extends>
   <header>BaseWidgets/ggColorWheelWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ggColorChannelWidget</class>
   <extends>QWidget</extends>
   <header>BaseWidgets/ggColorChannelWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mGraphicsView</tabstop>
  <tabstop>mOpenPushButton</tabstop>
  <tabstop>mSaveAsPushButton</tabstop>
  <tabstop>mNewCollectionPushButton</tabstop>
  <tabstop>mNewClassPushButton</tabstop>
  <tabstop>mNewClassBoxPushButton</tabstop>
  <tabstop>mNewFramePushButton</tabstop>
  <tabstop>mDeleteCollectionPushButton</tabstop>
  <tabstop>mDeleteClassPushButton</tabstop>
  <tabstop>mDeleteConnectionPushButton</tabstop>
  <tabstop>mDeleteClassBoxPushButton</tabstop>
  <tabstop>mDeleteFramePushButton</tabstop>
  <tabstop>mMoveTopPushButton</tabstop>
  <tabstop>mMoveUpPushButton</tabstop>
  <tabstop>mMoveDownPushButton</tabstop>
  <tabstop>mMoveBottomPushButton</tabstop>
  <tabstop>mAlignTL</tabstop>
  <tabstop>mAlignTC</tabstop>
  <tabstop>mAlignTR</tabstop>
  <tabstop>mAlignCL</tabstop>
  <tabstop>mAlignCC</tabstop>
  <tabstop>mAlignCR</tabstop>
  <tabstop>mAlignBL</tabstop>
  <tabstop>mAlignBC</tabstop>
  <tabstop>mAlignBR</tabstop>
  <tabstop>mZoomComboBox</tabstop>
  <tabstop>mZoomResetPushButton</tabstop>
  <tabstop>mZoomFitPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>SetZoom(QString)</slot>
 </slots>
</ui>
